docs/01-app/04-api-reference/05-config/01-next-config-js/serverComponentsHmrCache.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: serverComponentsHmrCache
+title: serverComponents&lt;wbr /&gt;HmrCache
```

Comment:
I think this gets read directly for metadata, so this might not work

Reply:
Hmm, im not actually sure, as its hooked into content layer